src/views/Curators/Curator.vue

```diff
@@ -0,0 +1,534 @@
+            let records = dataCuration['recentCuratorCreations'];
+            records.forEach(item => {
+              let object = {};
+              object.recordNameID = item.name+' ('+item.id+')';
+              object.createdAt = item.createdAt;
+              if (item['creator'] != null){
+                object.creator = item['creator'].username +' ('+item['creator'].id+')';
+              }else{
+                object.creator = "unknown"
+              }
+              _module.recordsCreatedCuratorsLastWeek.push(object);
+            });
+          },
+          prepareRecordsInCuration(dataCuration){
+            let _module = this;
+            let userRecords = dataCuration['recordsInCuration'];
+            userRecords.forEach(item => {
+              item['fairsharingRecords'].forEach(rec => {
+                let object = {};
+                object.curator = item['username'];
+                object.recordNameID = rec.name+' ('+rec.id+')';
+                let numMaint = 0;
+                object.recordMaintainers = "none"
+                rec['maintainers'].forEach(main => {
+                  if (numMaint > 0){
+                    object.recordMaintainers += ', ' + main.username+' ('+main.id+')';
+                  }else{
+                    object.recordMaintainers = main.username+' ('+main.id+')';
+                  }
+                  numMaint += 1;
+                });
+                _module.recordsInCuration.push(object);
+              });
+            });
+          },
+          prepareRecordsWithoutDois(dataCuration){
+            let _module = this;
+            let records = dataCuration['recordsWithoutDois'];
+            records.forEach(item => {
+              let object = {};
+              object.recordNameID = item.name+' ('+item.id+')';
+              object.createdAt = item.createdAt;
+              object.updatedAt = item.updatedAt;
+              if (item['creator'] != null){
+                object.creator = item['creator'].username +' ('+item['creator'].id+')';
+              }else{
+                object.creator = "unknown"
+              }
+              if (item['lastEditor'] != null){
+                object.lastEditor = item['lastEditor'].username +' ('+item['lastEditor'].id+')';
+              }else{
+                object.lastEditor = "unknown"
+              }
+              _module.recordsWithoutDois.push(object);
+            });
+          },
+          prepareHiddenRecords(dataCuration){
+            let _module = this;
+            let records = dataCuration['hiddenRecords'];
+            records.forEach(item => {
+              let object = {};
+              object.recordNameID = item.name+' ('+item.id+')';
+              object.createdAt = item.createdAt;
+              if (item.curator != null){
+                object.curator = item['curator'].username
+              }else{
+                object.curator = 'none'
+              }
+              if (item['creator'] != null){
```

Comment:
use the dot notation if the property does not have special characters (dashes, whitespace or else). You can also check for falsy values here rather than simple equality (!=) to `null` (You need to check also against `undefined` and `''`). You should almost never use simple equality.
```suggestion
              if (!item.creator){
```